compute_endpoint/globus_compute_endpoint/cli.py

```diff
@@ -1031,6 +1041,83 @@ def disable_on_boot_cmd(ep_dir: pathlib.Path):
     disable_on_boot(ep_dir)
 
 
+@app.command(
+    "render-user-config",
+    help="Render the user configuration template to stdout",
+)
```

Comment:
Let's make this easier for folks with a couple of examples.  Note that Click's `@app.command` decorator has an [epilog argument](https://click.palletsprojects.com/en/stable/documentation/#command-epilog-help); the `globus-idm-validator` uses this to give two different example invocations.